climada/util/coordinates.py

```diff
@@ -34,28 +34,30 @@
 from fiona.crs import from_epsg
 import geopandas as gpd
 import numpy as np
+from numba import jit
 import pandas as pd
+import pyproj
```

Comment:
This is a new dependency. Is it absolutely necessary? Has this been discussed with @emanuel-schmid . Please, as a general convention, before adding new dependencies in a pull request discuss it openly to see whether a) it is really needed b) it is compatible with the environment c) does not add many extra implicit dependencies.

Reply:
`pyproj` is not a new requirement for CLIMADA. It has been included in the requirements as an indirect dependency (through geopandas) for a long time and is used in several places in the code base. But maybe it's true that we should explicitly list it in the requirements file for the hypothetical case that geopandas might drop this dependency at some point in the future.

Reply:
Here are examples of other packages, that are explicitly imported in CLIMADA, but not listed in the requirements files: * cftime
* fiona
* mock
* netCDF4
* numpy
* pyproj
* shapefile
* shapely

Reply:
Good points! It seems to me that some of these requirements should really be explicitly listed in the requirements file.

I agree that the import in general does not necessarily need to be discussed here, but this is still exactly the goal of pull requests of this type: discuss the general architecture which includes the discussion about external dependencies.